src/app/services/api-data.service.ts

```diff
@@ -85,6 +108,27 @@ export class ApiDataService {
     );
   }
 
+  getAllOrganismsFromProject(project: string) {
+    const url = this.hostSetting.host + 'organism/_search/?size=100000&q=secondaryProject:' + project;
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```

Comment:
same thing here, headers is not required.

Reply:
done in c257a2a